Multistage Dockerfile build fails on Windows

As reported by [@hclbaur](https://github.com/hclbaur) building using the multistage Dockerfile build does not work on Windows due to carriage return versus line feed issues. See #44 .
Example of an error:
```
#13 196.1 [ERROR] Tests run: 10, Failures: 5, Errors: 0, Skipped: 0, Time elapsed: 56.805 s <<< FAILURE! - in io.axual.ksml.KSMLTopologyGeneratorBasicTest
#13 196.1 [ERROR] parseAndCheckOuput{int}[1]  Time elapsed: 55.643 s  <<< FAILURE!
#13 196.1 java.lang.AssertionError:
#13 196.1
#13 196.1 Expected: is "Topologies:\r\n   Sub-topology: 0\r\n    Source: sensor_stream (topics: [ksml_sensordata_avro])\r\n      --> for_each_001\r\n    Processor: for_each_001 (stores: [])\r\n      --> none\r\n      <-- sensor_stream\r\n\r\n"
#13 196.1      but: was "Topologies:\n   Sub-topology: 0\n    Source: sensor_stream (topics: [ksml_sensordata_avro])\n      --> for_each_001\n    Processor: for_each_001 (stores: [])\n      --> none\n      <-- sensor_stream\n\n"
#13 196.1       at io.axual.ksml.KSMLTopologyGeneratorBasicTest.parseAndCheckOuput(KSMLTopologyGeneratorBasicTest.java:76)
```

--- a/ksml/src/test/java/io/axual/ksml/KSMLTopologyGeneratorBasicTest.java
+++ b/ksml/src/test/java/io/axual/ksml/KSMLTopologyGeneratorBasicTest.java
@@ -24,6 +24,8 @@ package io.axual.ksml;
 import org.apache.kafka.streams.StreamsBuilder;
 import org.apache.kafka.streams.TopologyDescription;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 import org.junit.jupiter.params.provider.ValueSource;
@@ -84,13 +86,22 @@ public class KSMLTopologyGeneratorBasicTest {
             "leave short@123 alone,leave short@123 alone"
     })
     void cleanDescriptionTest(String input, String expected) {
+        System.out.println("input='" + input +"',expected='" + expected + "'");
         assertThat(cleanDescription(input), is(expected));
     }
 
+    @Test
+    @DisplayName("cleanDescription also fixes line endings")
+    void cleanDescriptionTestLineEnd() {
+        assertThat(cleanDescription("fix\r\nnewlines"), is("fix\nnewlines"));
+    }
+
     /**
-     * Clean a description string by removing all object references ("@abcd1234")
+     * Clean a description string by removing all object references ("@abcd1234") and fixing Windows line endings.
      */
     private String cleanDescription(String description) {
-        return description.replaceAll("@[a-fA-F-0-9]{5,}", "");
+        return description
+                .replaceAll("@[a-fA-F-0-9]{5,}", "")
+                .replaceAll("\r\n", "\n");
     }
 }

